src/growth-pandora.R

```diff
@@ -52,45 +62,46 @@ OutputOptionsSpatialPrometheus <- datasheet(myScenario, "burnP3PlusPrometheus_Ou
 fuelsRaster <- datasheet(myScenario, "burnP3Plus_LandscapeRasters")[["FuelGridFileName"]] %>% rast()
 elevationRaster <- tryCatch(
   datasheet(myScenario, "burnP3Plus_LandscapeRasters")[["ElevationGridFileName"]] %>% rast(),
-  error = function(e) NULL)
+  error = function(e) NULL
+)
 
 ## Handle empty values ----
-if(nrow(FuelTypeCrosswalk) == 0) {
+if (nrow(FuelTypeCrosswalk) == 0) {
   updateRunLog("No fuels code crosswalk found! Using default crosswalk for Canadian Forest Service fuel codes.", type = "warning")
   FuelTypeCrosswalk <- read_csv(file.path(ssimEnvironment()$PackageDirectory, "Default Fuel Crosswalk.csv")) %>% as.data.frame()
   saveDatasheet(myScenario, FuelTypeCrosswalk, "burnP3PlusPrometheus_FuelCodeCrosswalk")
 }
 
-if(nrow(OutputOptions) == 0) {
+if (nrow(OutputOptions) == 0) {
   updateRunLog("No tabular output options chosen. Defaulting to keeping all tabular outputs.", type = "info")
-  OutputOptions[1,] <- rep(TRUE, length(OutputOptions[1,]))
+  OutputOptions[1, ] <- rep(TRUE, length(OutputOptions[1, ]))
   saveDatasheet(myScenario, OutputOptions, "burnP3Plus_OutputOption")
 }
 
-if(nrow(OutputOptionsSpatial) == 0) {
+if (nrow(OutputOptionsSpatial) == 0) {
   updateRunLog("No spatial output options chosen. Defaulting to keeping all spatial outputs.", type = "info")
-  OutputOptionsSpatial[1,] <- rep(TRUE, length(OutputOptionsSpatial[1,]))
+  OutputOptionsSpatial[1, ] <- rep(TRUE, length(OutputOptionsSpatial[1, ]))
```

Comment:
I was going this way because of the concern that someone may miss it and it's easier to give them data as they can't get it once the run is done without re-running.